tox.ini

```
@@ -1,9 +1,9 @@
 [tox]
 minversion = 3.14.6
 envlist =
-    py{37,38,39,310}-django32-wagtail{41,42,50}-factoryboy32
-    py{38,39,310}-django40-wagtail{41,42,50}-factoryboy32
-    py{38,39,310,311}-django41-wagtail{41,42,50}-factoryboy32
+    py{38,39,310}-django32-wagtail{41,42,50,51}-factoryboy32
+    py{38,39,310}-django40-wagtail{41,42,50,51}-factoryboy32
+    py{38,39,310,311}-django41-wagtail{41,42,50,51}-factoryboy32
```

Kat came up earlier with a nicer matrix for us to use so It would be good to use this. Also as django 4.0 is obsolete it''d be good to drop it.

```suggestion
    py{38,39,310}-django{32,41}-wagtail{41,42,50,51}-factoryboy32
    py311-django41-wagtail{41,42,50,51}-factoryboy32
    ppy311-django42-wtagtail{50,51}-factoryboy32
```

---

Nice, looks good